src/server/graph-generation/interaction/index.js

```diff
@@ -19,23 +23,24 @@ function edgeType(type) {
   }
 }
 
-function rawGetInteractionGraphFromPC(interactionIDs){
+function rawGetInteractionGraphFromPC( interactionIDs ){
   const geneIds = _.uniq(_.concat([], interactionIDs)); //convert sources to array
 
   const params = {
-    cmd : 'graph',
+    directed: false,
+    limit: 1,
     source : geneIds,
-    pattern : ['controls-phosphorylation-of','in-complex-with','controls-expression-of', 'interacts-with'],
-    kind : geneIds.length > 1 ? 'pathsbetween' : 'neighborhood',
-    format : 'txt'
+    pattern : [ 'CONTROLS_PHOSPHORYLATION_OF','IN_COMPLEX_WITH','CONTROLS_EXPRESSION_OF', 'INTERACTS_WITH' ]
   };
 
-  //Fetch graph from PC
-  return pc.query(params).then(res => {
+  const url = ( geneIds.length > 1 ? pathsbetweenUrl : neighborhoodUrl ) + qs.stringify( params );
+  return fetch(url)
```

Comment:
All you should be doing is changing the params object to pc.query.

Reply:
unless you've refactored much there, pc.query uses PC_URL and has nothing to do with the new service (sifgraph); we in fact want to use both PC2 (/get, /traverse, /graph) and Sifgraph endpoints in the app.

Reply:
The pc module can accept any command and query options, right?  They're just parameters.  The concern with using a separate fetch, as this PR does, is that it's duplicated code and it doesn't accept the PC URL env var.

I think we should be using the existing pc module with a command like 'sifgraph/v1/neighborhood'.

Reply:
Yeah, I somewhat agree with Dylan and Max; but in order to make it work with like commands cmd:'sifgraph/v1/neighborhood', we have to change the meaning and default value of PC_URL - exclude "pc2/" from it - and do some refactoring (e.g., find/replace 'get' with 'pc2/get', 'search' with 'pc2/search', etc.) through app-ui code.